src/guild/config/guildConfigs.ts

```diff
@@ -197,9 +197,18 @@ guildConfigs.set('546414872196415501', {
 		},
 		censor: {
 			inviteWhitelist: INVITE_WHITELIST,
-			allowSwearing: false,
```

Comment:
Is this false by default since you removed it entirely?

Reply:
Yeah, swearing will be disallowed by default if `allowSwearing` is omitted